### PR TITLE
Hooks ⚡️

### PR DESCRIPTION
**Pull request by SaraVieira**
## Description - [Trello ticket](put a link to the trello ticket here)

put a short summary of what you did here

## Review template

- [x] checked that this PR resolves the spec provided from trello / this description
- [x] ensured that this PR does not introduce any obvious bugs

### CODE REVIEW

**Comment by SaraVieira** [comment]
YAY! More code removed then added

File: src/components/Common/GridDebugger.js [hunk 1]
```diff
@@ -33,44 +33,36 @@ const ColumnContent = styled.div`
...
-    window.addEventListener('keyup', this.handleKey)
-  }
+const GridDebugger = () => {
+  const [showGrid, toggleShowGrid] = useState(false)
 
-  componentWillUnmount() {
-    window.removeEventListener('keyup', this.handleKey)
+  const handleKey = ev => {
+    if (ev.key === 'g' && ev.ctrlKey) {
+      console.log('lol')
```
Comment (antoniocapelo) [inline src/components/Common/GridDebugger.js]: unneeded lol

Comment (SaraVieira) [reply src/components/Common/GridDebugger.js]: unneeded? unneeded? FRIENDSHIP OVER

Comment (antoniocapelo) [reply src/components/Common/GridDebugger.js]: 😢

Comment (SaraVieira) [reply src/components/Common/GridDebugger.js]: Removed

**Pull request merged**